apiary.apib

```diff
@@ -1545,6 +1545,77 @@ Lists tables in a given bucket.
                 }
             ]
 
+## Create Typed Table [/v2/storage/buckets/{bucket_id}/tables-definition]
+
+### Create new table definition [POST]
+
+
++ Parameters
+    + bucket_id (required) - The bucket Id
+
++ Request (application/json)
+    + Headers
+
+            X-StorageApi-Token: your_token
+
+    + Body
+
+        {
+            "name": String,
```

Comment:
V dokumentaci opravit jednu chybu ktoru vyhodi apirary.
plus by som popisal tie atributy JSONu tak ako to ma napr Create Bucket (bohuzial teraz neviem z hlavy nejaky JSON api ktory mame takto popisany)
Upravit ten request tak aby ked si zavolam curl s ukazky na apiary tak je validny a mozem ho zavolat